docs/index.html

```diff
@@ -241,7 +242,7 @@ <h3>Accelerate </h3>
 							</div>
 							<div class="fh5co-post to-animate">
 								<h3>Customized API</h3>
-								<p>BiiLabs APIs based on IOTA Tangle enable your service like Digital ID, ledger & settlement, digital forensics, micro-payment, etc. on blockchain. We provide and secure sandbox for testing and make your service on blockchain would be super easy.</p>
+								<p>Based on IOTA Tangle, BiiLabs realizes the service such as Digital ID, ledger & settlement, micro-payment, digital forensics, etc. by both providing and securing sandbox for testing, which means the customization of API.</p>
```

Comment:
Is it appropriate to use "realize"?

Reply:
perhaps "implement the service" would be better